apps/opentelemetry/include/otel_span.hrl

```
@@ -18,48 +18,48 @@
 
 -record(span, {
           %% 128 bit int trace id
-          trace_id                :: opentelemetry:trace_id() | undefined,
+          trace_id                :: opentelemetry:trace_id() | undefined | '_',
```

ugh yeah this whole file's annotations are sort of annoying. I wish there was a way for records to be annotated as match-spec/select-spec friendly.

---

oooh, maybe there is a better way. I commented in the gradualizer discussion () about if its a possibility to just ignore those. Since its only in a matchspec it should even be possible to not end up ignoring errors if for some reason a user is actually use those values in their record and not just in match specs.